src/settings-ui/Settings.UI/Views/GeneralPage.xaml

```diff
@@ -369,8 +369,13 @@
                             </labs:SettingsCard>
                         </labs:SettingsExpander.Items>
                     </labs:SettingsExpander>
+                    <labs:SettingsCard x:Uid="GeneralPage_EnableExperimentation"
+                                       HeaderIcon="{ui:BitmapIcon Source=/Assets/FluentIcons/FluentIconsExperimentation.png}">
+                        <ToggleSwitch
+                            x:Uid="ToggleSwitch"
+                            IsOn="{Binding Mode=TwoWay, Path=EnableExperimentation}" />
```

Comment:
[off-topic] Why use this syntax over `x:Bind EnableExperimentation ...` ?

Reply:
I replicated the code I saw for previous toggle switches in the same file. 
https://github.com/microsoft/PowerToys/blob/af70b6b6897054be73e5e6ffb9b585ac3d3a264a/src/settings-ui/Settings.UI/Views/GeneralPage.xaml#L221-L223